examples/simple/flake.nix

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Could you show me an example of a full machine deployment config, including system closure, a user profile, and a service profile?

Reply:
In `examples/system`